app/services/attendance_csv_importer.rb

```diff
@@ -17,7 +17,9 @@ def initialize
     @archive_bucket = Rails.application.config.aws_necc_attendance_archive_bucket
   end
 
-  def call
+  def call(start_date = nil, end_date = 0.days.after)
```

Comment:
rather than set these in the call block, I'd rather we do it in the initialize and leave the call block pristine, so it'd be like AttendanceCsvImporter.new(start_date, end_date).call

Also - can you use named params instead of positional ones?  (i.e. `def initialize(start_date: nil, end_date: 0.days.after)`